docs/spacefinder.md

```diff
@@ -0,0 +1,85 @@
+# Spacefinder
+
+Spacefinder identifies slots in articles where ads can be inserted dynamically. Every paragraph is assessed against a set of rules, with 'winning' candidates having ads placed above them.
+
+- [How Spacefinder works](#how-spacefinder-works)
+- [Spacefinder rules](#spacefinder-rules)
+- [Style and layout of dynamic ads](#style-and-layout-of-dynamic-ads)
+
+Ideally Spacefinder hits the sweet spot between high ad ratio for us and a peerless reading experience for _Guardian_ website visitors.
```

Comment:
According to [the style guide](https://www.theguardian.com/guardian-observer-style-guide-t#:~:text=lc%20for%20newspapers%20(-,the%20Guardian,-)%2C%20magazines%20(the%20New):

> lc for newspapers (the Guardian), magazines (the New Statesman), pubs (the Coach and Horses), bands (the Black Eyed Peas, the Not Sensibles, the The), nicknames (the Hulk, the Red Baron), and sports grounds (the Oval).

```suggestion
Ideally Spacefinder hits the sweet spot between high ad ratio for us and a peerless reading experience for _the Guardian_ website visitors.
```

Reply:
Ah good spot. You've made me realise the(!) Guardian doesn't seem to italicise publication names. [Here for example](https://www.theguardian.com/news/2021/oct/03/pandora-papers-biggest-ever-leak-of-offshore-data-exposes-financial-secrets-of-rich-and-powerful). I can't find an official line on this, though [the style guide entry on italics](https://www.theguardian.com/guardian-observer-style-guide-i) does share this nugget of wisdom from George Bernard Shaw:

> "1. I was reading The Merchant of Venice. 2. I was reading ‘The Merchant of Venice’. 3. I was reading **The Merchant of Venice**. The man who cannot see that No 1 is the best-looking, as well as the sufficient and sensible form, should print or write nothing but advertisements for lost dogs or ironmongers’ catalogues: literature is not for him to meddle with."